packages/main/src/themes/Carousel.css

```diff
@@ -122,19 +126,19 @@
 }
 
 .ui5-carousel-navigation-dot {
-    width: var(--ui5_carousel_width);
-    height: var(--ui5_carousel_height);
-    margin: var(--ui5_carousel_margin);
+    width: var(--ui5_carousel_inactive_dot_width);
+    height: var(--ui5_carousel_inactive_dot_height);
```

Comment:
A circle has the same dimension for height and width. You can contract these two variables into one variable - something like `ui5_carousel_inactive_dot_size`.